test/TestPkg/src/TestPkg.jl

```diff
@@ -0,0 +1,183 @@
+module TestPkg
```

Comment:
I fully agree to include the tests you have implemented here. I suggest to include few comments (perhaps outside from `TestPkg` simply to describe what you are testing. Our memory is finite and we are prone to forget!